scylla/src/transport/topology.rs

```diff
@@ -21,7 +24,7 @@ pub struct MetadataReader {
     control_connection_address: SocketAddr,
     control_connection: NodeConnectionPool,
 
-    // when control connection fails, TopologyReader tries to connect to one of known_peers
+    // when control connection fails, MetadataReader tries to connect to one of known_peers
```

Comment:
Nit: there seem to be some leftover renames in this commit which should be included in the first commit.